deps/quicly/lib/quicly.c

```diff
@@ -5509,14 +5501,14 @@ static int handle_ack_frequency_frame(quicly_conn_t *conn, struct st_quicly_hand
     int ret;
 
     /* recognize the frame only when the support has been advertised */
-    if (!recognize_delayed_ack(conn))
+    if (conn->super.ctx->transport_params.min_ack_delay_usec == UINT64_MAX)
         return QUICLY_TRANSPORT_ERROR_FRAME_ENCODING;
 
     if ((ret = quicly_decode_ack_frequency_frame(&state->src, state->end, &frame)) != 0)
         return ret;
 
     QUICLY_PROBE(ACK_FREQUENCY_RECEIVE, conn, conn->stash.now, frame.sequence, frame.packet_tolerance, frame.max_ack_delay,
-                 frame.ignore_order);
+                 (int)frame.ignore_order, (int)frame.ignore_ce);
 
     /* At the moment, the only value that the remote peer would send is this value, because our TP.min_ack_delay and max_ack_delay
      * are equal. */
```

Comment:
Peers can advertise `frame.max_ack_delay` of 25ms. We can ignore them (pointed out to @janaiyengar).

Reply:
This issue was addressed in https://github.com/h2o/quicly/pull/491.